Makefile

```diff
@@ -0,0 +1,15 @@
+# Package configuration
```

Comment:
Can you please add a header comment to this file, outlining how it's meant to be used? 

I think the answer is just "run `make` and don't look back at the explosion", but it's helpful to know that this is meant to be used _inside_ the container (I think), rather than by someone's local work tree.

Reply:
there's a generic template of Makefile for [CI](https://github.com/src-d/ci/tree/v1) 

Done.